kickstart/kickstart.json

```diff
@@ -54,11 +54,12 @@
               "application": {
                 "name": "Example app",
                 "oauthConfiguration" : {
-                    "authorizedRedirectURLs": ["http://localhost:3000","http://localhost"],
-                    "authorizedOriginURLs": ["http://localhost:9000","http://localhost"],
+                    "authorizedRedirectURLs": ["http://localhost:3000","http://localhost:9000/app/token-exchange"],
+                    "authorizedOriginURLs": ["http://localhost:9000","http://localhost:3000"],
                     "clientSecret": "super-secret-secret-that-should-be-regenerated-for-production",
                     "logoutURL": "http://localhost:3000",
                     "enabledGrants": ["authorization_code","refresh_token"],
+                    "debug": true,
```

Comment:
We could additionally add the following to the kickstart to set up cors:

`{
          "method": "PATCH",
          "url": "api/system-configuration",
          "body":{
            "systemConfiguration": {
              "corsConfiguration": {
                "allowCredentials": true,
                "allowedMethods": [
                  "GET",
                  "POST",
                  "OPTIONS"
                ],
                "allowedOrigins": [
                  "http://localhost:3000"
                ],
                "debug": false,
                "enabled": true,
                "preflightMaxAgeInSeconds": 0
              }
            }
          }
        }`

Reply:
Nice.  done.